### DISCUSSION TITLE
想要在bpmn:definitions中添加新的属性用于适配flowable

### DISCUSSION THREAD

**Discussion by cfc3058353263** [didi/LogicFlow#1407] [General]
![image](https://github.com/didi/LogicFlow/assets/52273637/866e8ae2-d7bf-423c-92ef-1ba1109c537e)
使用该方法不生效了

**Comment by wumail** [comment]
什么版本有效，什么版本无效？

**Reply by cfc3058353263** [reply]
![image](https://github.com/didi/LogicFlow/assets/52273637/0fb6b9ed-0fc5-4343-ae46-b11642d782ba)
我把依赖更新到最新版本试过还是没有效果

**Reply by wumail** [reply]
之前什么版本有效？

**Reply by cfc3058353263** [reply]
"@logicflow/core": "^1.2.1",
 "@logicflow/extension": "^1.2.1",

**Reply by wumail** [reply]
实测"@logicflow/extension": "1.2.17"下该配置有效，建议在codesandbox中复现一下，判断是否是logicflow的问题

**Reply by cfc3058353263** [reply]
![image](https://github.com/didi/LogicFlow/assets/52273637/15c5ecec-07f8-4be4-9eb9-c94c5147eaad)
将这个注释掉之后就可以了，这里的写法有哪里不对吗

**Reply by wumail** [reply]
你用的应该不是同一个插件，一个是bpmnXmlAdapter一个是BPMNAdapter，你试试给`lf.extension.BPMNAdapter.definitionAttributes`赋值，我试了是OK的，最好是使用BPMNAdapter，bpmnXmlAdapter后续不一定会再维护了

**Reply by cfc3058353263** [reply]
好的，谢谢
